LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/the-stylepropertymap/properties/background-size-expected.txt

```diff
@@ -9,7 +9,7 @@ PASS Can set 'background-size' to a length: -3.14em
 PASS Can set 'background-size' to a length: 3.14cm
 PASS Can set 'background-size' to a length: calc(0px + 0em)
 PASS Can set 'background-size' to a percent: 0%
-PASS Can set 'background-size' to a percent: -3.14%
+FAIL Can set 'background-size' to a percent: -3.14% assert_approx_equals: expected -3.14 +/- 0.000001 but got 0
```

Comment:
Same question here, what's the expected result here?

Reply:
I believe this test is wrong. background-size is defined to be "<bg-size> = [ <length-percentage [0,∞]> | auto ]{1,2} | cover | contain", so this would not roundtrip due to non-negative requirement.